glib/gobject-sys/src/lib.rs

```diff
@@ -1869,6 +1895,55 @@ extern "C" {
     pub fn g_binding_get_target_property(binding: *mut GBinding) -> *const c_char;
     pub fn g_binding_unbind(binding: *mut GBinding);
 
+    //=========================================================================
```

Comment:
Any reason for not generation those new APIs?

Reply:
It needs manual bindings, just like for single signal/property bindings.

Reply:
Unfortunate i guess, something for later

Reply:
It's quite low level API :)